interop-switch/templates/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.ingress.enabled -}}
+{{- $serviceName := include "interop-switch.fullname" . -}}
+{{- $servicePort := .Values.service.externalPort -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "interop-switch.fullname" . }}
+  labels:
+    app: {{ template "interop-switch.name" . }}
```

Comment:
We are using fullname for the app label.
Please change too: app: {{ template "interop-switch.fullname" . }}